src/fiatconnect/saga.ts

```diff
@@ -997,13 +998,26 @@ export function* handleCreateFiatConnectTransfer(
   const quoteId = fiatConnectQuote.getQuoteId()
   let transactionHash: string | null = null
   try {
-    const { transferAddress }: TransferResponse = yield call(_initiateTransferWithProvider, action)
+    const { transferAddress, transferId }: TransferResponse = yield call(
+      _initiateTransferWithProvider,
+      action
+    )
 
     if (flow === CICOFlow.CashOut) {
-      transactionHash = yield call(_initiateSendTxToProvider, {
+      const cashOutTxHash: string = yield call(_initiateSendTxToProvider, {
```

Comment:
Why not just keep the assignment to `transactionHash` here? We won't have one for cash-ins anyways, so don't think we need to differentiate here.